modelci/app/experimental/endpoints/cv_tuner.py

```diff
@@ -131,7 +143,7 @@ def update_finetune_model_as_new(id: str, updated_layer: Structure, dry_run: boo
             framework=model.framework,
             engine=Engine.NONE,
             model_status=[ModelStatus.DRAFT],
-            version=ModelVersion(model.version.ver + 1),
+            version=version,
```

Comment:
Note that you may reconsider the primary key constraint on the model.

Reply:
the model has a primary key on (name, framework, engine, version). This won't allow the user to have multiple model draft version.

Reply:
Yes, these lines of code are trying to avoid model saving error due to duplicate draft version